src/courtroom.cpp

```diff
@@ -4408,6 +4408,8 @@ void Courtroom::on_reload_theme_clicked()
 
   enter_courtroom();
   update_character(m_cid);
+  if (music_player->is_playing)
+    ui_music_display->play("music_display");
```

Comment:
This does not fix the problem... The actual issue is that the play() play_once option is reset somewhere that makes it disappear after the first loop.